src/Data/List/Relation/Binary/Permutation/Propositional.agda

```diff
@@ -68,6 +87,28 @@ data _↭_ : Rel (List A) a where
   { isEquivalence = ↭-isEquivalence
   }
 
+------------------------------------------------------------------------
+-- _↭_ is equivalent to `Setoid`-based permutation
```

Comment:
I don't think that's going to work for precisely the reasons that distinguish the use of *intensional* aspects of the definitions form their *extensional ones*. In particular, I don't think it will even hold without further binding in the specific constructor definitions for the two relations. 

In any case, as this PR is intended as the first part of a longer sequence intended to conclude with a resolution of both #1354 and #2311 , I'm inclined to say this review comment as mission-creep that's out-of-scope at this stage?

Reply:
Ok.